microsoft-integration/media-gallery-tool/src/main/resources/Messages_eu.properties

```diff
@@ -0,0 +1,52 @@
+index_title=Microsoft Media Gallery
```

Comment:
Hey @pablosanroman, this is a new tool, would you mind providing a translation for these texts? Thanks!